build_library/grub_install.sh

```diff
@@ -192,6 +196,26 @@ case "${FLAGS_target}" in
         sudo cp "${BUILD_LIBRARY_DIR}/menu.lst" \
             "${ESP_DIR}/boot/grub/menu.lst"
         ;;
+    arm64-efi)
+        info "Installing default arm64 UEFI bootloader."
+        sudo mkdir -p "${ESP_DIR}/EFI/boot"
+        #FIXME(andrejro): shim not ported to aarch64
+        # Use the test keys for signing unofficial builds
+#        if [[ ${COREOS_OFFICIAL:-0} -ne 1 ]]; then
```

Comment:
Just leave out the unused code, it can easily be pasted in later. :) Leaving the FIXME is fine though